### PR TITLE
ONBOARDING: User Interface completed

### PR DESCRIPTION
**Pull request by priettt**
Finished activity_main.xml with the design of the calculator

Signed-off-by: Francisco Prieto francisco.prieto@globant.com

### CODE REVIEW

**Comment by priettt** [comment]
33f17e39-eb84-4573-bb70-5606479076fb

Here's the screenshot of the calculator!

**Review by fanky10 — APPROVED** [review]
Please pay more attention to avoiding hardcoded values in xml. It looks good though

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -1,39 +1,184 @@
...
-            android:text="@string/calculator_title"
-            android:layout_marginTop="@dimen/margin_all_normal_24"
-            app:layout_constraintTop_toTopOf="parent"
-            app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintEnd_toEndOf="parent"/>
+        android:id="@+id/text_calculator_visor"
+        android:layout_width="match_parent"
+        android:layout_height="@dimen/calculator_visor_height"
+        android:background="@color/material_red_400"
+        android:elevation="4dp"
```
Comment (SartiMau) [inline app/src/main/res/layout/activity_main.xml]: Use dimens please

File: app/src/main/res/layout/activity_main.xml [hunk 2]
```diff
@@ -1,39 +1,184 @@
...
-        android:layout_height="match_parent"
-        tools:context=".MainActivity">
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".MainActivity">
 
     <TextView
```
Comment (SartiMau) [inline app/src/main/res/layout/activity_main.xml]: Use Styles for the styles attributes

File: app/src/main/res/layout/activity_main.xml [hunk 3]
```diff
@@ -1,39 +1,184 @@
...
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toStartOf="@id/button_divide"
+        app:layout_constraintStart_toEndOf="@id/button_0"
+        app:layout_constraintTop_toBottomOf="@+id/button_3" />
+
+    <Button
+        android:id="@+id/button_backspace"
+        style="@style/Button.Calculator.Operation"
+        android:textSize="@dimen/text_calculator_number_button"
+        android:layout_marginStart="1dp"
```
Comment (SartiMau) [inline app/src/main/res/layout/activity_main.xml]: Use dimens please

**Review by SartiMau — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/activity_main.xml (hunk 1 above)
Comment (priettt) [reply app/src/main/res/layout/activity_main.xml]: done!

File: app/src/main/res/layout/activity_main.xml (hunk 2 above)
Comment (priettt) [reply app/src/main/res/layout/activity_main.xml]: done!

File: app/src/main/res/layout/activity_main.xml (hunk 3 above)
Comment (priettt) [reply app/src/main/res/layout/activity_main.xml]: done!

**Review by SartiMau — APPROVED** [review]

**Review by fanky10 — APPROVED** [review]
LG2M

**Pull request merged**